lmdeploy/pytorch/engine/devices/cuda.py

```diff
@@ -0,0 +1,12 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+from .base_device_utils import BaseDeviceUtils
+
+
+class CUDADeviceUtils(BaseDeviceUtils):
+
+    device = 'cuda'
```

Comment:
The class name indicates CUDA. Is "device = 'cuda' " necessary?

Reply:
https://github.com/InternLM/lmdeploy/blob/41f179ee4812d310132a107db011ec4a13d46670/lmdeploy/pytorch/engine/devices/base_device_utils.py#L18

The class would be registered automatically with this key when being imported.